control-plane-operator/controllers/hostedcontrolplane/oauth/deployment.go

```diff
@@ -211,3 +212,22 @@ func buildOAuthVolumeProvidersTemplate(v *corev1.Volume) {
 		SecretName: manifests.OAuthServerDefaultProviderSelectionTemplateSecret("").Name,
 	}
 }
+
+func applyNamedCertificateMounts(certs []configv1.APIServerNamedServingCert, spec *corev1.PodSpec) {
+	container := &spec.Containers[0]
```

Comment:
Can we select the container by name so we won't break this if we ever add another container here?

Reply:
Yep we can do that